contracts/protocol/libraries/logic/ValidationLogic.sol

```diff
@@ -687,10 +691,10 @@ library ValidationLogic {
   }
 
   /**
-   * @notice Validates if an asset can be activated as collateral in the following actions: supply, transfer,
-   * set as collateral, mint unbacked, and liquidate
+   * @notice Validates if an asset can be activated as collateral in the following actions: set as collateral, mint unbacked
    * @dev This is used to ensure that the constraints for isolated assets are respected by all the actions that
```

Comment:
This @dev feels outdated now

Reply:
not sure what you mean, i think that's still accurate no? `validateAutomaticUseAsCollateral` just puts some things on top

Reply:
the doc says " @dev This is used to ensure that the constraints for isolated assets are respected by all the actions that generate transfers of aTokens"
This function only validates constrains for 2 functions

Reply:
I mean it implicitly does as `validateAutomaticUseAsCollateral` will call this function as well, but sure can change. Will do once the other issues are concluded.